### ISSUE TITLE
Old bulk api document

### ISSUE THREAD

**Issue by ftomassetti** [LionWeb-io/specification#242]
I think that [this document](https://github.com/LionWeb-io/specification/blob/main/bulk/repo-access-api.adoc) is outdated and could be removed

**Comment by enikao** [comment]
I don't think so -- that's the current spec (also published at https://lionweb.io/specification/bulk/repo-access-api.html )
